src/planscape/forsys/views.py

```diff
@@ -271,6 +290,73 @@ def generate_project_areas_for_a_single_scenario(
         return HttpResponseBadRequest("Ill-formed request: " + str(e))
 
 
+@csrf_exempt
+def generate_project_areas_prototype(
```

Comment:
nit: can this specify it's the prototype for a post-lambda call?

There are two endpoints - one before lambda, and one after lambda, and it'd be great to clarify that.

Reply:
renamed to generate_project_areas_from_lambda_output_prototype